### ISSUE TITLE
nice to have: handle "test data being deleted" message nicely

### ISSUE THREAD

**Issue by isaacs** [tierrun/tier#46]
Not sure how many tier users other than you and me actually would be affected but this, but it'd be nice if this message was reported in something prettier than plain old json

```
failed	plan:name@title0		https://dashboard.stripe.com/test/products/tier_plan_name_title0	[{"status":409,"message":"Your test data is in the process of being deleted. Once this is complete you will be able to make test mode requests again.","request_id":"req_jwAIp0vLfTqumh","type":"invalid_request_error"}]
tier: {"status":409,"message":"Your test data is in the process of being deleted. Once this is complete you will be able to make test mode requests again.","request_id":"req_jwAIp0vLfTqumh","type":"invalid_request_error"}
```

(Maybe, we could just always print the errors somewhat nicer?  Something like `error ${type}: ${message} (status=${status})` or something?  They seem to always have the same basic shape, if it's anything else, just dump it out.)
